src/Map/Turn.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using ChessDemo.Entities;
+
+namespace ChessDemo.Map
+{
+    class Turn
+    {
+        /// List of Teams to cycle through
+        private List<Team> teamList;
+
+        /// Dictionary of teams to entities
+        private Dictionary<Entity, Team> teamDict;
+
+        /// Team which is currently active
+        public Team currentTeam;
+
+        public Turn(Dictionary<Entity, Team> teams)
```

Comment:
With TurnManager being a singletong, make an Add(Dictionary<Entity, Team> team). Probably will also want Add(Entity, Team) so we can add something to a Team later. I think TurnManager can be the decisive factor for who is on what team.